README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# Student Introduction: Elliot Karl
+### MACS 30500 HW Assignment #1
+### October 5, 2019
+
+####
+**About Me:** My name is Elliot Karl and I am a rising second year Masters of Public Policy Student (MPP) at the Harris School of Public Policy. I enrolled in *MACS 30500: Computing for Social Sciences* because I am hoping to:
+
+1. Build on the introductory knowledge I gained in using R at Harris
+2. Gain confidence in programming in R so that I can
+i) transform large budget data sets for future work in the local public sector which had become tedious (or impossible) in excel,
```

Comment:
Would be great if these sublists are actually show as sublists with indentation. You would want to do that by adding a tab in front of each sublist